cmd/speep/main.go

```diff
@@ -98,15 +127,23 @@ func HandleGet(getCmd *flag.FlagSet, all *bool, name *string) error {
 }
 
 func ValidateNewShortcutKey(addCmd *flag.FlagSet, name *string, key *string) error {
-	addCmd.Parse(os.Args[2:])
 	if *name == "" || *key == "" {
 		addCmd.PrintDefaults()
 		return errors.New("name and the shortcut key are required to add a shortcut key")
 	}
 	return nil
 }
 
-func HandleAdd(addCmd *flag.FlagSet, name *string, newShortcut *string) error {
+func HandleAdd(addCmd *flag.FlagSet, name *string, nameShort *string, newShortcut *string, newShortcutShort *string) error {
```

Comment:
This function is now taking too many parameters IMO -- consider creating a struct to hold these parameters and reduce your function parameters down to just 1.

Same applies for HandleDelete